### ISSUE TITLE
Error running tests right after adapter creation with typescript

### ISSUE THREAD

**Issue by watho** [AlCalzone/create-adapter#1084]
Adapter creation is successful with dev-server running and showing the new adapter, but running the tests leads to an failure.

## step to reproduce

1. Successful: npx @iobroker/create-adapter@latest
2. Successful: npm run build
3. Successful: dev-server run
4. npm run test:ts
leads to 
```
> iobroker.adapter-create-test@0.0.1 test:ts
> mocha --config test/mocharc.custom.json src/**/*.test.ts


✖ ERROR: Error: Cannot find module 'tsx/cjs'
Require stack:
- /tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/nodejs/esm-utils.js
- /tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/mocha.js
- /tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/cli/one-and-dones.js
- /tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/cli/options.js
- /tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/bin/mocha.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at exports.requireOrImport (/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/nodejs/esm-utils.js:53:16)
    at async exports.handleRequires (/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/cli/run-helpers.js:94:28)
    at async /tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/cli/run.js:349:25 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/nodejs/esm-utils.js',
    '/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/mocha.js',
    '/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/cli/one-and-dones.js',
    '/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/lib/cli/options.js',
    '/tmp/adaptertest/ioBroker.adapter-create-test/node_modules/mocha/bin/mocha.js'
  ]
}

```

**Comment by AlCalzone** [comment]
Ah damn, I must have replaced one occurrence of "ts-node" too many.

Replace "tsx/cjs" with "ts-node/register" as a workaround.

**Comment by watho** [comment]
Thx. Replacing it in ````test/mocharc.custom.json```` works for me.

**Comment by AlCalzone** [comment]
fixed by #1086
